deployment/monocular/templates/ui-config.yaml

```diff
@@ -13,6 +13,9 @@ data:
       overrides: {
         googleAnalyticsId: '{{.Values.ui.googleAnalyticsId}}',
         appName: '{{.Values.ui.appName}}',
+        {{- if .Values.ui.backendHostname }}
+        backendHostname: '{{ .Values.ui.backendHostname }}',
```

Comment:
I think it makes sense to remove it because we can use a subdomain like `api.mydomain.com`. Adding `/api` may be repetitive. However, I think this is not so important now and it's a different issue.